doc/Language/variables.pod6

```diff
@@ -343,9 +342,9 @@ sub say-it()  { say $^a; } # invalid
 -> $x, $y, $x { say $^a; } # invalid
 =end code
 
-Placeholder variables syntactically cannot have any type constraints. Be
-also aware that one cannot have placeholder variables with a single
-upper-case letter. This is disallowed in favor of being to able to catch
+Placeholder variables syntactically cannot have any type constraints.
+Placeholder variabless cannot have a single
```

Comment:
How about:
Placeholder variables cannot have type constraints or a variable name with a single upper-case letter (this is disallowed to enable catching some Perl 5-isms).